src/convertors/roam/roam-convertor.test.ts

```diff
@@ -425,4 +426,11 @@ describe('RoamConvertor', () => {
 
     expect(notes).toMatchSnapshot()
   })
+
+  it('handles missing uid issue', async () => {
```

Comment:
Why it's missing *uid*? I guess tt should be missing *title* based on the typescript interface changes

Reply:
That test is badly named